test/pystorm/test_component.py

```diff
@@ -272,6 +238,38 @@ def test_read_tuple(self):
             tup = component.read_tuple()
             self.assertEqual(output, tup)
 
+    def test_read_tuple_named_fields(self):
+        # This is only valid for bolts, so we only need to test with task IDs
+        # and Tuples
```

Comment:
It's a bit odd to have a feature in `Component` which is only for bolts, but I understand that this is in `Component` for ... reasons.